receiver/azureblobreceiver/factory.go

```diff
@@ -108,8 +112,14 @@ func (f *blobReceiverFactory) getReceiver(
 			return nil
 		}
 
+		var blobEventHandler BlobEventHandler
+		blobEventHandler, err = f.getBlobEventHandler(receiverConfig, set.Logger)
+		if err != nil {
+			return nil
```

Comment:
This error should not be ignored
```suggestion
			return fmt.Errorf("failed to build blob event handler: %w", err)
```

Reply:
It is not. It is returned [here](https://github.com/eedorenko/opentelemetry-collector-contrib/blob/62a097ef57844fb67e1e170831af4c807beb41ea/receiver/azureblobreceiver/factory.go#L127)